data/src/main/java/com/puzzlebench/cmk/data/model/CharacterRealm.kt

```diff
@@ -2,5 +2,10 @@ package com.puzzlebench.cmk.data.model
 
 import io.realm.RealmObject
 
-
-open class CharacterRealm(var name: String? = null, var description: String? = null, var thumbnail: ThumbnailRealm? =null) : RealmObject()
\ No newline at end of file
+open class CharacterRealm(
```

Comment:
please do not initialize variables in NULL in kotlin, only do that when is necessary

Reply:
done